content/ja/handbook/engineering/development/fulfillment/index.md

```
@@ -0,0 +1,851 @@
+
+個々のグループは [プロダクト開発フロー](/handbook/product-development/how-we-work/product-development-flow/#workflow-summary) ワークフローのステージをそれぞれの用途に応じて自由に使用できますが、Issue が発見/精査から実装へと移行する方法については、ある程度規定する必要があります。
+
+### ユーザーエクスペリエンス
+
+私たちはすべてのワークフローで優れた使いやすさを提供し、ユーザーとビジネスのニーズのバランスを取ることを目指しています。プロダクトデザイナーはプロダクトマネージャーとエンジニアと密接に連携します。
+
+#### 作業方法
+
+- ハンドブックの [プロダクトデザイン](/handbook/product/ux/) セクションに記載されている [プロダクトデザイナーワークフロー](/handbook/product/ux/product-designer/) および [UX リサーチャーワークフロー](/handbook/product/ux/experience-research/) に従います。
+- [UX スコアカード](/handbook/product/ux/ux-scorecards/) を使用して進捗を測定します。
+- 四半期ごとに最も重要なプロジェクトを優先し、Fulfillment プロダクトデザイナーは [グループではなくプロジェクトをサポート](#product-designer-focus-areas) します。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)